account_bank_statement_import_save_file/tests/test_save_file.py

```diff
@@ -16,20 +16,23 @@ class HelloWorldParser(models.TransientModel):
 
     @api.model
     def _parse_file(self, data_file):
-        return (
-            'EUR',
-            acc_number,
-            [{
-                'name': '000000123',
-                'date': '2013-06-26',
-                'transactions': [{
-                    'name': 'KBC-INVESTERINGSKREDIET 787-5562831-01',
+        if self.env.context.get('test_save_file'):
```

Comment:
the underlying problem here is that the parser model is not cleaned up after the test. That's also difficult to do, so let's not start with it. But instead of the context key (if you really need this, better namespace with your module name), why don't you analyze the content of the file? That actually could for other modules be extended that for tests, we use a file that contains the module's name. That's simple to detect then.

Reply:
Thanks for the review! Time is always the problem. I will change it as you suggest as soon as possible.